### PR TITLE
Task 4 Nodemon

### PR DESCRIPTION
**Pull request by Himanshu0k**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "nodemon": "^3.1.7"
```
Comment (GargPermil) [inline package.json:4]: need configuration as well.

Comment (Himanshu0k) [reply package.json:4]: added.

**Review by GargPermil — CHANGES_REQUESTED** [review]
remove node modules.

**Comment by Himanshu0k** [comment]
removed.

**Pull request merged**